catalog.html

```diff
@@ -195,7 +203,7 @@ <h3 class="product-card-title">GreenTree Inn</h3>
           <li class="pagination-item">
             <a class="pagination-link" href="#">4</a>
           <li class="pagination-item">
-            <a class="visually-hidden" href="">...</a>
+            <span class="visually-hidden" href="">...</span>
```

Comment:
```suggestion
            <span class="visually-hidden">...</span>
```
`href` бывает только у ссылки.